bundles/core/org.eclipse.smarthome.core.audio.test/src/test/groovy/org/eclipse/smarthome/core/audio/test/AudioConsoleTest.groovy

```diff
@@ -62,6 +63,30 @@ public class AudioConsoleTest extends AudioOSGiTest {
         assertCompatibleFormat()
     }
 
+    @Test
+    public void 'audio console plays file for a specified sink with a specified volume'(){
```

Comment:
In the light of https://github.com/eclipse/smarthome/issues/3305, please do not add new Groovy tests, but rather help migrating the few existing ones in here to Java - thanks!

Reply:
I keep this in mind and submit a follow up PR.